test.sh

```diff
@@ -23,8 +15,12 @@ if [ ! -z "${2}" ]; then
 fi
 
 dlHttpFiles() {
-  dir=${WORKSPACE}/build/static/http/common
-  if [ "${REPO_NAME}" != "on-http" ]; then
+  if [ -n "${MULTI}" ]; then
```

Comment:
I didn't find $MULTI  in your test job ?

how about add some comment to help review ?